chef-config/lib/chef-config/config.rb

```diff
@@ -634,7 +634,7 @@ def self.target_mode?
     # Path to the default CA bundle files.
     default :ssl_ca_path, nil
     default(:ssl_ca_file) do
-      if ChefUtils.windows? && embedded_dir
+      if embedded_dir
```

Comment:
so why wasn't this done on windows to begin with?  where did this code come from and is there any rationale in the original PR?  and what effect is this going to have on unix users?

Reply:
This code was added https://github.com/chef/chef/commit/eecb3f3cdbe1eb4e77794f33340a6690e49b9d34

In Unix/Linux machine the certificate is present under `/opt/chef/embedded/ssl/certs/cacert.pem`

So earlier it was set default configuration of `ssl_ca_file` only for Windows, and for Unix/Linux machine it was set as nil.

Reply:
@mwrock @lamont-granquist Can we close it or anything we can do if required, please put your suggestion so that i can proceed.

Reply:
My question is what is this fixing. I know it aims to "fix" #10073 but I'm not sure that is a bug but just an implementation detail. Do we have customer scenarios where 10073 actually break chef runs on non-windows platforms? If not, I think we should close.

Reply:
@mwrock @lamont-granquist @tas50 since we are not getting any chef failure so I think we can close PR.